src/shared/Microsoft.Git.CredentialManager.Tests/Commands/UnconfigureCommandTests.cs

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+using System.Threading.Tasks;
+using Microsoft.Git.CredentialManager.Commands;
+using Microsoft.Git.CredentialManager.Tests.Objects;
+using Moq;
+using Xunit;
+
+namespace Microsoft.Git.CredentialManager.Tests.Commands
+{
+    public class UnconfigureCommandTests
+    {
+        [Theory]
+        [InlineData("unconfigure", true)]
+        [InlineData("UNCONFIGURE", true)]
+        [InlineData("uNcOnFiGuRe", true)]
```

Comment:
What you've done for configure/unconfigure seems great, I don't know if we need to support all cases (upper/lower chars), but the work seems minimal and have great tests so what you have here is definitely fine by me.